README.md

```diff
@@ -87,3 +87,65 @@ The controller can be configured using cmd args:
       --watch-all-namespaces                      Watch for resources in all namespaces, if set to false it will only watch the runtime namespace. (default true)
       --watch-label-selector string               Watch for resources with matching labels e.g. 'sharding.fluxcd.io/shard=shard1'.
 ```
+
+## Usage
+
+### Basic Example
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: neo4j-aura-api
+  namespace: default
+data:
+  clientID: <base64-encoded-client-id>
+  clientSecret: <base64-encoded-client-secret>
+---
+apiVersion: neo4j.infra.doodle.com/v1beta1
+kind: AuraInstance
+metadata:
+  name: my-neo4j-instance
+  namespace: default
+spec:
+  tier: professional-db
+  region: eu-central-1
+  cloudProvider: aws
+  neo4jVersion: "5"
+  tenantID: 928f3731-1111-5ffd-a2f7-3602aafb304b
+  memory: 8GB
+  secret:
+    name: neo4j-aura-api
+```
```

Comment:
Isn't this pretty much what the quickstart above already presents? 
[SecretReference](https://github.com/DoodleScheduling/neo4j-aura-controller?tab=readme-ov-file#quickstart)